Dockerfile

```
@@ -10,11 +10,15 @@ RUN useradd $username -u $user_id --create-home --user-group
 
 COPY ./dist/suzieq-$version-py3-none-any.whl  /tmp/
 
-RUN pip install --upgrade pip
-RUN pip install /tmp/suzieq-$version-py3-none-any.whl
-RUN rm /tmp/suzieq-$version-py3-none-any.whl
 COPY --chown=$username suzieq/config/etc/suzieq-cfg.yml /home/$username/.suzieq/suzieq-cfg.yml
-RUN sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml
+
+RUN python3 -m pip install --upgrade pip && \
+    python3 -m pip install /tmp/suzieq-$version-py3-none-any.whl && \
+    rm /tmp/suzieq-$version-py3-none-any.whl && \
+    mkdir /home/$username/parquet && \ 
+    sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml
```

```suggestion
RUN python3 -m pip install --upgrade pip

COPY --chown=$username suzieq/config/etc/suzieq-cfg.yml /home/$username/.suzieq/suzieq-cfg.yml

RUN mkdir /home/$username/parquet && \
    chown $user_id:$user_id /home/$username/parquet && \
    sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml

COPY ./dist/suzieq-$version-py3-none-any.whl  /tmp/

RUN python3 -m pip install /tmp/suzieq-$version-py3-none-any.whl && \
    rm /tmp/suzieq-$version-py3-none-any.whl
```
I agree about reducing the number of layer, but I would split the RUNs in order to use caching, so reducing build time and memory usage. For example, we don't need to update pip again if the wheel changes.

Additionally probably you forgot to chown parquet directory.